src/coreComponents/dataRepository/Group.hpp

```diff
@@ -323,7 +333,12 @@ class Group
   T & getGroup( KEY const & key )
   {
     Group * const child = m_subGroups[ key ];
-    GEOSX_THROW_IF( child == nullptr, "Group " << getPath() << " doesn't have a child " << key, std::domain_error );
+    if( child == nullptr )
+    {
+      GEOSX_THROW( "Group " << getPath() << " has no child named " << key << std::endl <<
```

Comment:
use GEOSX_FMT instead of streams ?

Reply:
I personally think that this would nest a `GEOSX_FMT` into a `GEOSX_THROW`, and that `GEOSX_FMT` forces to move back and forth the lines. I guess the added value of `fmt` is better formatting, possibility to reorder/reuse arguments, share error message templates under the form of shared strings. If any of this does not apply, I'm not sure it adds a lot to use `GEOSX_FMT`.